redox_recipe.sh

```diff
@@ -0,0 +1,10 @@
+GIT=https://github.com/RustPython/RustPython
+BRANCH=redox
```

Comment:
Consider moving the redox related scripts into a seperate folder.

Reply:
I could actually probably just embed the recipe into the install script, do you think I should do that?

Reply:
No, probably not. Having the recipe seperate is good.